.github/workflows/check-documentation.yml

```diff
@@ -21,6 +21,12 @@ jobs:
         uses: actions/setup-go@v5
         with: { go-version-file: go.mod }
 
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: "1.10.5"
```

Comment:
I personally wouldn't pin it so that we don't need to maintain it 🤷 I'd let it break

Reply:
Yeah, I'm typically on the opposing side, I prefer breaking stuff rather than maintaining :)

Reply:
You like to embrace the dark side 🥷